internal/network/cidr_validations.go

```diff
@@ -35,7 +27,11 @@ func VerifyCIDRsNotOverlap(aCidrStr, bCidrStr string) error {
 	if err != nil {
 		return err
 	}
-	return verifyCIDRsNotOverlap(acidr, bcidr)
+	//overlapping occur if one of the CIDRs is a subset of the other
```

Comment:
Why is this change needed ?

Reply:
Why having so many sub functions with similar names? The 2 liners were used just here so I merged them. I found it clearer to understand as a reader and follow the flow